client/my-sites/subscriptions/index.tsx

```diff
@@ -1,10 +1,20 @@
 import config from '@automattic/calypso-config';
+import { LocaleProvider, i18nDefaultLocaleSlug } from '@automattic/i18n-utils';
 import SubscriptionManager from '@automattic/subscription-manager';
 import { useTranslate } from 'i18n-calypso';
 import page, { Callback } from 'page';
 import { createElement } from 'react';
+import MomentProvider from 'calypso/components/localized-moment/provider';
 import { makeLayout, render } from 'calypso/controller';
 
+declare global {
```

Comment:
I think this might be added to `clients/types.ts`.

Reply:
True, thanks for the feedback. I changed the implementation removing this `declare global` and the `LocaleProvider` in favor of `useLocale` hook, which takes the locale from the right place (instead of me getting it from globals).

Reply:
Nice, I'll test again!

Reply:
Thank you!

Reply:
Works as intended!